.gitignore

```diff
@@ -189,3 +189,10 @@ infer/src/.project
 /infer/src/deadcode/*.mli
 _opam
 _coverage
+
+# website related
+website/translated_docs/
```

Comment:
Let's remove `website/translated_docs/` and `website/i18n/*` first since it's not being used yet.

Reply:
Done